server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -197,10 +211,50 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 	echo json_encode($response);
 }
 
+function storeConsent($reviewConsent, $virtualConsent, $signature, $appointmentId) {
+	GLOBAL $DB_CONN;
+
+	$response = array();
+	$response['success'] = false;
+	
+	if($virtualConsent === 'true' && $virtualConsent !== null && $signature !== null && $signature !== '' && $appointmentId !== null) {
+		$reviewConsent = $reviewConsent === 'true' ? 1 : 0;
+		$virtualConsent = $virtualConsent === 'true' ? 1 : 0;
+		try {
+			$DB_CONN->beginTransaction();
+
+			insertConsent($reviewConsent, $virtualConsent, $signature, $appointmentId);
+			$DB_CONN->commit();
```

Comment:
Since there's actually only one insertion here, you don't need the beginTransaction/commit--you would only do that if you were inserting multiple times and every single one had to complete successfully for the DB to be in a correct state. Given that there's only one insertion here, we're guaranteed to be in a correct state since there's nothing this could rely on